docs/Admin.md

```diff
@@ -533,6 +533,33 @@ const App = () => (
 );
 ```
 
+## `notification`
+
+You can override the notification component, for instance to change the notification duration. It defaults to 4000, i.e. 4 seconds, and you can override it using the `autoHideDuration` prop. For instance, to create a custom Notification component with a 5 seconds default:
```

Comment:
```suggestion
You can override the notification component, for instance to change the notification duration. A common use case is to change the `autoHideDuration`, and force the notification to remain on screen longer than the default 4 seconds. For instance, to create a custom Notification component with a 5 seconds default:
```

Reply:
This was taken from the recipes section, should I change it there too?